cli.go

```diff
@@ -16,6 +16,7 @@ type CLIOptions struct {
 	AssumeRoleARN  string            `help:"the ARN of the role to assume" default:"" env:"ECSPRESSO_ASSUME_ROLE_ARN"`
 	Timeout        *time.Duration    `help:"timeout. Override in a configuration file." env:"ECSPRESSO_TIMEOUT"`
 	FilterCommand  string            `help:"filter command" env:"ECSPRESSO_FILTER_COMMAND"`
+	NoColor        bool              `help:"disables colorized output" default:"false"`
```

Comment:
I think the negative flag that has a default false value is confusing a little.

alecthomas/kong has the `negatable` attribute to make `--no-xxx` flag from the positive flag `--xxx` automatically.

```suggestion
	Color          bool              `help:"enable colorized output" env:"ECSPRESSO_COLOR" default:"true" negatable:""`
```

Reply:
Fixed in https://github.com/kayac/ecspresso/commit/aee93f243be33d56b229febc2334f4a2d8921967 .

Thanks for the suggestion！